src/main.rs

```diff
@@ -160,6 +164,72 @@ fn parse_matches() -> clap::ArgMatches {
                 .default_value("latest")
                 .help("Docker image tag."),
         )
+        // Bootstrap/Validator Config
+        .arg(
+            Arg::with_name("tpu_enable_udp")
+                .long("tpu-enable-udp")
+                .help("Validator config. Enable UDP for tpu transactions."),
+        )
+        .arg(
+            Arg::with_name("tpu_disable_quic")
+                .long("tpu-disable-quic")
+                .help("Validator config. Disable quic for tpu packet forwarding"),
+        )
+        .arg(
+            Arg::with_name("limit_ledger_size")
+                .long("limit-ledger-size")
+                .takes_value(true)
+                .help("Validator Config. The `--limit-ledger-size` parameter allows you to specify how many ledger
+                shreds your node retains on disk. If you do not
+                include this parameter, the validator will keep the entire ledger until it runs
+                out of disk space. The default value attempts to keep the ledger disk usage
+                under 500GB. More or less disk usage may be requested by adding an argument to
+                `--limit-ledger-size` if desired. Check `agave-validator --help` for the
+                default limit value used by `--limit-ledger-size`. More information about
+                selecting a custom limit value is at : https://github.com/solana-labs/solana/blob/583cec922b6107e0f85c7e14cb5e642bc7dfb340/core/src/ledger_cleanup_service.rs#L15-L26"),
```

Comment:
nit: how about using a default value of `DEFAULT_MAX_LEDGER_SHREDS.to_string()`? that way you can just `unwrap()` fetching this later